Marlin/Configuration_adv.h

```diff
@@ -1536,11 +1536,14 @@
    * We encourage you to take advantage of this new feature and we also
    * respectfully request that you retain the unmodified Marlin boot screen.
    */
-  #define SHOW_BOOTSCREEN                 // Show the Marlin bootscreen on startup. ** ENABLE FOR PRODUCTION **
-  #if ENABLED(SHOW_BOOTSCREEN)
-    #define BOOTSCREEN_TIMEOUT 4000       // (ms) Total Duration to display the boot screen(s)
-    #if EITHER(HAS_MARLINUI_U8GLIB, TFT_COLOR_UI)
-      #define BOOT_MARLIN_LOGO_SMALL      // Show a smaller Marlin logo on the Boot Screen (saving lots of flash)
+  #if ANY(HAS_MARLINUI_U8GLIB, TOUCH_UI_FTDI_EVE, IS_DWIN_MARLINUI)
```

Comment:
This now (re)excludes boot screens from text/2004 LCDs.

Reply:
We both had a pretty close look at the changes that this is reverting. If there are some screens that don't yet respect the `SHOW_BOOTSCREEN` setting, it would be better for @classicrocker883 to get jiggy with it and apply the setting to those displays.

Reply:
In https://github.com/MarlinFirmware/Marlin/issues/25763#issuecomment-1537076688 they said:

> it simply would not boot with `SHOW_BOOTSCREEN` or `BOOTSCREEN_TIMEOUT`

This isn’t in the case in the ProUI fork / before, so it seems fixes should be elsewhere.

Reply:
I'll do one more test just to confirm the issue in build, I did use Mriscoc's direct source repo. alas, I will "get jiggy with it" like @thinkyhead says 👍

Reply:
after another quick test I can confirm this is a false alarm. it could have been something else after I merged in the update after those changes. so disregard that config_adv change.